setuptools/dist.py

```diff
@@ -361,6 +357,20 @@ def __init__(self, attrs=None):
                 )
         self._finalize_requires()
 
+    def _install_setup_requires(self, attrs):
+        if attrs.pop('parse_config', False):
```

Comment:
Since this parameter is private and not meant for any user to invoke, it should probably be called `_parse_config` or similar.